src/main/java/com/limechain/network/protocol/warp/DigestHelper.java

```diff
@@ -37,4 +40,13 @@ public static Optional<BabePreDigest> getBabePreRuntimeDigest(HeaderDigest[] hea
                 .map(HeaderDigest::getMessage)
                 .map(message -> ScaleUtils.Decode.decode(message, new PreDigestReader()));
     }
+
+    public HeaderDigest buildSealHeaderDigest(BlockHeader blockHeader, Schnorrkel.KeyPair keyPair) {
```

Comment:
You can make ```buildSealHeaderDigest``` static, and there will be no need to make an instance of the class to use it.